### ISSUE TITLE
Change internal memory path

### ISSUE THREAD

**Issue by XRedCubeX** [TeamWin/Team-Win-Recovery-Project#1568]
- [NO] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [YES] I am running the latest version of TWRP
- [YES] I have read the FAQ (https://twrp.me/FAQ/)
- [YES] I have searched for my issue and it does not already exist

**Device codename**: <E506plus>
**TWRP version**: <3.3.1>

#### WHAT STEPS WILL REPRODUCE THE PROBLEM?
<Go to the file manager and go to /sdcard>

#### WHAT IS THE EXPECTED RESULT?
<Going to the file manager and then to the /sdcard, the internal memory should be displayed correctly>

#### WHAT HAPPENS INSTEAD?
<The internal memory is not displayed because TWRP recognizes that the internal memory is located in /data/media/0 but in reality this device has the folder in /data/media>

#### ADDITIONAL INFORMATION
<The only way to fix it would be to move the / sdcard directory from /data/media/0 to /data/media />

Device Tree: https://github.com/XRedCubeX/twrp_device_E506plus

<https://paste.omnirom.org/view/79f2fc23>

**Comment by bigbiff** [comment]
My guess is that you are moving around in android versions? The 0 directory was added when multi-user was added to Android. Can you remove the 0 folder manually and see if that fixes your issue? TWRP doesn't make any assumptions other than /data/media/0 exists then use that as sdcard.

**Comment by XRedCubeX** [comment]
No, my device only has android 5.1 and does not have the multiuser (strange😅), when I delete the directory nothing changes.

**Comment by bigbiff** [comment]
After deleting the directory, can you restart recovery and upload the recovery.log?

**Comment by XRedCubeX** [comment]
I solved with the "RECOVERY_SDCARD_ON_DATA: = true" flag, I also made sure that at every start the TWRP must delete the 0 folder in / sdcard, however every time the ROM is successfully started and then restarts in recovery, the folder is recreated and must be deleted manually, it is not a problem as much because I will have to make a lollipop rom for that device and it will definitely not have the bug
